packages/web-app-external/src/components/LoadingScreen.vue

```diff
@@ -4,3 +4,9 @@
     <p v-translate class="oc-invisible">Loading app</p>
   </div>
 </template>
+
+<script>
+export default {
```

Comment:
Just added the script part to give the component a name and let my IDE give the imports pretty colors.